src/iexamine.cpp

```diff
@@ -681,25 +683,30 @@ void iexamine::cardreader(player &p, const tripoint &examp)
                 g->remove_zombie( critter );
             }
         }
-        add_msg(_("You insert your ID card."));
-        add_msg(m_good, _("The nearby doors slide into the floor."));
-        p.use_amount(card_type, 1);
+        if( open ) {
+            add_msg( _( "You insert your ID card." ) );
+            add_msg( m_good, _( "The nearby doors slide into the floor." ) );
+            p.use_amount( card_type, 1 );
+            //return;
```

Comment:
Why is this code commented out?

Reply:
Oops, my bad, forgot to remove the commented code.